src/pages/JoinPage/JoinPage.tsx

```diff
@@ -21,7 +20,6 @@ import {
 } from './queries';
 
 export const JoinPage = () => {
-  useAuthStateMachine(false);
```

Comment:
curious if this still all works the same but not super important this is not in our critical path

Reply:
todo: test this